pandas/tests/window/test_ewm.py

```diff
@@ -181,3 +181,14 @@ def test_ewma_times_adjust_false_raises():
         Series(range(1)).ewm(
             0.1, adjust=False, times=date_range("2000", freq="D", periods=1)
         )
+
+
+@pytest.mark.parametrize("func", ["mean", "std", "var"])
+@pytest.mark.parametrize("dtype", [np.float32, np.float16, np.float64, float, "float"])
```

Comment:
Could you use the `float_dtype` pytest fixture instead?

Reply:
I don't see a `float_dtype` fixture. Are you suggesting to add that?

Reply:
My bad. Added.